SP23.P03.Web/Data/SeedHelper.cs

```diff
@@ -202,4 +211,56 @@ private static async Task AddPassengers(IServiceProvider serviceProvider, DataCo
 
         await dataContext.SaveChangesAsync();
     }
+
+    private static async Task AddTrips(DataContext dataContext)
+    {
+        var trips = dataContext.Set<Trip>();
+
+        if (await trips.AnyAsync())
+        {
+            return;
+        }
+
+        var offset = TimeZoneInfo.Local.BaseUtcOffset;
+
+        var stations = dataContext.Set<TrainStation>();
+        var hammondStation = await stations.FirstOrDefaultAsync(x => x.Name == "Hammond Station");
+        var slidellStation = await stations.FirstOrDefaultAsync(x => x.Name == "Slidell Station");
+        var nolaStation = await stations.FirstOrDefaultAsync(x => x.Name == "NOLA Station");
+
+        var trains = dataContext.Set<Train>();
+        var hammondTrain = await trains.FirstOrDefaultAsync(x => x.Name == "Hammond Train");
```

Comment:
```suggestion
        var hammondStation = await stations.FirstAsync(x => x.Name == "Hammond Station");
        var slidellStation = await stations.FirstAsync(x => x.Name == "Slidell Station");
        var nolaStation = await stations.FirstAsync(x => x.Name == "NOLA Station");

        var trains = dataContext.Set<Train>();
        var hammondTrain = await trains.FirstAsync(x => x.Name == "Hammond Train");
```
Giving these the possibility of returning null causes a few warnings